azext_iot/tests/test_iot_central_int.py

```diff
@@ -220,6 +220,25 @@ def test_central_device_methods_CRD(self):
 
         self._delete_device(device_id)
 
+    def test_central_user_methods_CRD(self):
+        users = self._create_users()
+
+        self.cmd(
+            "iot central app user show --app-id {} --user-id {}".format(
+                APP_ID, users[0].get("id")
+            ),
+        )
+
+        result = self.cmd(
+            "iot central app user list --app-id {}".format(APP_ID,),
+        ).get_output_in_json()
+
+        user_list = result.get("value")
+
+        for user in users:
+            assert user in user_list
+            self._delete_user(user.get("id"))
```

Comment:
nit - it might be worth doing the delete operations before doing asserts. Since we already have the list, we can verify it whenever we want

```
for user in users:
    self._delete_user(user["id"]) # no need for get, also if `id` is missing then it should catastrophically fail the IT
```

```
for user in users:
    assert user in user_list
```